doc/classes/InputEventMIDI.xml

```diff
@@ -1,32 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="InputEventMIDI" inherits="InputEvent" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Represents an input event from a MIDI device, such as a piano.
+		Represents a MIDI message from a MIDI device, such as a musical keyboard.
 	</brief_description>
 	<description>
-		InputEventMIDI allows receiving input events from MIDI (Musical Instrument Digital Interface) devices such as a piano.
-		MIDI signals can be sent over a 5-pin MIDI connector or over USB, if your device supports both be sure to check the settings in the device to see which output it's using.
-		To receive input events from MIDI devices, you need to call [method OS.open_midi_inputs]. You can check which devices are detected using [method OS.get_connected_midi_inputs].
+		InputEventMIDI stores information about messages from [url=https://en.wikipedia.org/wiki/MIDI]MIDI[/url] (Musical Instrument Digital Interface) devices. These may include musical keyboards, synthesizers, and drum machines.
+		MIDI messages can be received over a 5-pin MIDI connector or over USB. If your device supports both be sure to check the settings in the device to see which output it is using.
+		By default, Godot does not detect MIDI devices. You need to call [method OS.open_midi_inputs], first. You can check which devices are detected with [method OS.get_connected_midi_inputs], and close the connection with [method OS.close_midi_inputs].
 		[codeblocks]
 		[gdscript]
 		func _ready():
 		    OS.open_midi_inputs()
 		    print(OS.get_connected_midi_inputs())
 
-		func _input(input_event):
```

Comment:
I prefer to use `input_event` because 1. It matches the type and 2. `event` is a special keyword in some languages like C#, so `input_event` or `inputEvent` is more portable.